components/resources/ai_chat_ui_strings.grdp

```diff
@@ -42,4 +39,13 @@
   <message name="IDS_CHAT_UI_PAGE_CONTENT_WARNING" desc="Description about page content being sent to a remote LLM">
      Disconnect to stop sending this page content to Leo, and start a new conversation
   </message>
+  <message name="IDS_CHAT_UI_ERROR_NETWORK" desc="An error presented when there is a network issue in the UI">
+     There was a network issue connecting to Leo, check your connection and try again.
+  </message>
+  <message name="IDS_CHAT_UI_ERROR_RATE_LIMIT" desc="An error presented when the user has exhausted the API request limit">
+     You've reached the maximum number of questions for Leo. Please try again in a few hours.
```

Comment:
@petemill — the one thing I might note is that saying "answering questions" might imply some sort of "connection" between this user's chat and that user's chat. I dunno…like some kind of connectivity btwn chat windows. Also the interactions in Leo won't always be around questions.

Here are a few benign options:

`Oops. It looks like Leo is busy right now. Please try again in a few minutes.`
`It looks like Leo is busy helping other Brave users. Please try again in a few minutes.`
`It looks like Leo is backed up with questions from other Brave users. Please try again in a few minutes.`

And, if we want to get cute:

`It looks like Leo stepped away for some coffee. Please try again in a few minutes.`

Reply:
@rmcfadden3 ok so, since it _might_ be due to the user using Leo too much within the last hour / day or it _might_ be because there are too many other users using Leo at the moment, I would suggest your more generalised option: `It looks like Leo is busy right now. Please try again in a few minutes.`

Reply:
The "stepped away for some coffee" option seems better for the general error issue.

Reply:
the user may think that wording is even because the server is having a problem. We need to be pretty specific here that a usage limit was hit as we will modify the message later to provide an upgrade path here too.

Reply:
Following from this then, does it make sense to make it slightly more straightforward but still ambiguous as to whether the rate limit is local or global (since we won't know)? Right now it could be:

> Leo is too busy right now. Please try again in a few minutes.

And when there is an upgrade path, something like:

> Leo is too busy right now. Please try again in a few minutes or subscribe to Leo Premium for priority access.

Reply:
Copy changed to: "Leo is too busy right now. Please try again in a few minutes."